### ISSUE TITLE
On start show the current working directory

### ISSUE THREAD

**Issue by stokito** [elfmz/far2l#599]
When I run the far2l from command line I would like to see the current folder opened in far.

**Comment by stokito** [comment]
Ok, I opened sources of the `main.cpp` and found that the far2l actually have a builtin help but instead of `far --help` it should be called with `far2l /?`. And among all the options we have the option `cd`:

     /cd <path> Change panel's directory to specified path.

So to open a specified folder we can run `far2l /cd /path/to/folder`. 
For example:

    far2l /cd ~/Downloads
    far2l /cd /opt/

But what is interesting is that this doesn't work for the root folder `/` i.e. when I run `far2l /cd /` it again opens my home folder. I guess that's because the `/` symbol is used as the program args delimiter.

Which means that to open the current folder we can run `far2l cd ./` 
But for some reason it doesn't work like this.
Accidentally I runned far2l with the `cd` option instead of  `/cd` i.e. `far2l cd` and I opened the current folder. I tried to run again the `far2l cd` inside of root folder but it opened my home folder.
So here is few issues:
1. Migrate far2l to linux/gnu options wich uses a hyphen e.g. `-h` and `--help`
2. The `/cd` option is not working for root folder
3. The `cd` option without `/` is not documented
3. The `cd` option without `/` option opens current working directory if the path is not specified and this should be documented. Or...
5. Open the current working folder by default.
